hotsos/core/plugins/openstack/common.py

```diff
@@ -214,6 +214,20 @@ def apache2_certificates_expiring(self):
 
         return apache2_certificates_expiring
 
+    @cached_property
+    def apache2_allow_encoded_slashes_on(self):
+        """ Returns True if AllowEncodedSlashes On is found. """
+        if not self.ssl_enabled:
+            return False
+
+        with open(self.apache2_ssl_config_file) as fd:
+            for line in fd:
+                ret = re.search(r'AllowEncodedSlashes On /(\S+)', line)
+                if ret:
+                    return True
+
+            return False
```

Comment:
Probably better to intend his return at `with` level so that it returns `False` if open fails too.